content/en/esdc-c19-research.html

```diff
@@ -0,0 +1,108 @@
+---
+type: section
+layout: page
+title: "Take part in our research"
+url: /esdc-c19-research/
+---
+<section class="page--outer-container-padding">
+    <div class="row">
+        <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+            <h2 class="section--title">Confused about COVID-19 financial help in Canada?</h2>
+        
+            <p>Help us solve your worries. Help us create clear, understandable information about what help is available, eligibility criteria and estimated payments.</p>
+
+            <p>If you would like to help us, or know someone else who can, please contact <a href="mailto:adrianne.lee@tbs-sct.gc.ca">adrianne.lee@tbs-sct.gc.ca</a> for more information on how to be involved.</p>
+
+            <hr>
+
+            <h2>How you can help</h2>
+
+            <p>We are working with Employment and Social Development Canada (ESDC) to <b>create clear, understandable information about what help is available, eligibility criteria and estimated payments</b>.</p>
+            
+            <p>To do this we will <b>ask you to ask you to describe your situation and then try using a draft web page to see if the web page is helpful</b>. This will take <b>30 minutes</b>.</p>
```

Comment:
```suggestion
            <p>To do this we will <b>ask you to join an online Zoom video-conferencing session to describe your situation to us and then try using a draft web page to see if the content is helpful</b>. This will take <b>30 minutes</b>.</p>
```